elfo-core/src/lib.rs

```diff
@@ -76,6 +76,8 @@ mod supervisor;
 mod telemetry;
 mod thread;
 
```

Comment:
Why another mod's group (empty line between `mod`s)?

Reply:
Initially I thought they're kinda unrelated, fixed in [1070425](https://github.com/elfo-rs/elfo/pull/133/commits/1070425102a5c11c57e2f0617259ec6dbfb6c18c)